skimage/measure/fit.py

```diff
@@ -515,7 +528,11 @@ def estimate(self, data):
         C1 = np.array([[0., 0., 2.], [0., -1., 0.], [2., 0., 0.]])
 
         # Reduced scatter matrix [eqn. 29]
-        M = np.linalg.inv(C1).dot(S1 - np.dot(S2, np.linalg.inv(S3)).dot(S2.T))
+        try:
+            M = np.linalg.inv(C1).dot(
+                S1 - np.dot(S2, np.linalg.inv(S3)).dot(S2.T))
+        except:  # LinAlgError: Singular matrix
```

Comment:
well, the are two points, it anything happens during slowing this matrix multiplication, goes to not proper ellipse fitting and then the estimate fails anyway... second, I found this instability while testing ransac on real data so isolating the special case is possible but not common... :)